content/docs/guides/branching-github-actions.md

```diff
@@ -0,0 +1,117 @@
+      parent_id: br-long-forest-224191
+      branch_name: from_action_reusable
+      api_key: {{ secrets.NEON_API_KEY }}
+    id: create-branch
+  - run: echo project_id ${{ steps.create-branch.outputs.project_id}}
+  - run: echo branch_id ${{ steps.create-branch.outputs.branch_id}}
+```
+
+### Input variables
+
+- `project_id`: The ID of your Neon project. You can find this value in the Neon Console, on the **Settings** page.
+- `parent_branch_id`: The ID of the parent branch, typically the `main` branch of your project. You can find this value in the Neon Console. Select **Branches** from the sidebar, and then select the branch. A branch ID has a `br-` prefix.
+- `branch_name`: This is an optional parameter. If unspecified, the branch name will default to the same value as the `branch_id` of the newly created branch, which is a generated value that starts with a `br-` prefix.
+- `api_key`: An API key created in your Neon account. For instructions, see [Create an API key](/docs/manage/api-keys#create-an-api-key).
+
+### Outputs
+
+```yaml
+outputs:
+  branch_id:
+    description: "Newly created branch Id"
+    value: ${{ steps.output-branch-id.outputs.branch_id }}
+  project_id:
+    description: "Project Id"
+    value: ${{ steps.output-project-id.outputs.project_id }}
+```
+
+- `branch_id`: The ID of the newly created branch.
+- `project_id`: The ID of the parent branch.
+
+## Delete branch action
+
+This GitHub Action deletes a branch from your Neon project.
+
+<Admonition type="info">
+The source code for this action is available on [GitHub](https://github.com/neondatabase/delete-branch-action).
+</Admonition>
+
+### Prerequisites
+
+- Using the action requires a Neon API key. For information about obtaining an API key, see [Create an API key](/docs/manage/api-keys#create-an-api-key).
+- Add your Neon API key to your GitHub Secrets:
+    1. In your GitHub repository, go to **Settings** and locate **Secrets** at the bottom of the left sidebar.
+    2. Click **Actions** > **New Repository Secret**.
+    3. Name the secret `NEON_API_KEY` and paste your API key in the **Secret** field
+    4. Click **Add Secret**.
+
+### Example
+
+The following example deletes a branch with the `br-long-forest-224191` branch ID from a Neon project with the project ID `rapid-haze-373089` when a pull request is merged.
+
+```yaml
+name: Delete Neon Branch with GitHub Actions Demo
+run-name: Delete a Neon Branch 🚀
+on: [push]
+jobs:
+  delete-neon-branch:
+    uses: neondatabase/delete-branch-action@v3
+    with:
+      project_id: rapid-haze-373089
+      branch_id: br-long-forest-224191
+      api_key: {{ secrets.NEON_API_KEY }}
+```
+
+### Input variables
+
+- `project_id`: The ID of your Neon project. You can find this value in the Neon Console, on the **Settings** page.
+- `branch_id`: The ID of the branch you want to delete. Select **Branches** from the sidebar, and then select the branch. A branch ID has a `br-` prefix.
```

Comment:
```suggestion
- `branch:` The ID of the branch you want to delete. Select **Branches** from the sidebar, and then select the branch. A branch ID has a `br-` prefix.
```

it's not possible to use "branch" option to pass either branch id or branch name